mls-rs-uniffi/src/lib.rs

```diff
@@ -531,6 +555,15 @@ impl Group {
         group.write_to_storage().await.map_err(Into::into)
     }
 
+    /// Export the current epoch's ratchet tree in serialized format.
+    ///
+    /// This function is used to provide the current group tree to new
+    /// members when `use_ratchet_tree_extension` is set to false in
+    /// `ClientConfig`.
+    pub fn export_tree(&self) -> RatchetTree {
+        self.inner().export_tree().into_owned().into()
+    }
```

Comment:
It'll probably resolve itself with #120 but FYI this will clone the tree (which may get very large), while in most cases it's better to instead borrow the tree here and serialize it to bytes without cloning.

Reply:
Ah, thanks for the heads up!

Reply:
This should be fixed now.